fix: LEAP-364: Fix crosshair functionality

robot-ci-heartex opened the pull request:
Hi @Gondragos!

This PR was [created](https://github.com/HumanSignal/label-studio/actions/runs/6850633925) in a response to PRs in upstream repos:
- https://github.com/HumanSignal/label-studio-frontend/pull/1616

Gondragos committed:
[submodules] Build static HumanSignal/label-studio-frontend

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/6850633925

Gondragos committed:
Merge remote-tracking branch 'origin/develop' into fb-leap-364/fix-crosshair

# Conflicts:
#	label_studio/frontend/dist/lsf/js/main.js
#	label_studio/frontend/dist/lsf/js/main.js.map
#	label_studio/frontend/dist/lsf/version.json

Gondragos commented:
/help
> Command | Description
> --- | ---
> /frontend [\<args\> ...] | Actions with frontend. Type `/frontend help` for an additional help.

Gondragos commented:
/frontend help
> Command | Description
> --- | ---
> /frontend build           | Get build frontend static for all repos
> /frontend build lsf [sha] | Get build frontend static for HumanSignal/label-studio-frontend only
> /frontend build dm [sha]  | Get build frontend static for HumanSignal/dm2 only

Gondragos commented:
/frontend build lsf
> Successfully pushed new changes
>  heartexlabs/label-studio-frontend@4891f397096fe3a8773ec98ba7e56d0289810b82
>
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/6867199311)

Gondragos committed:
[frontend] Get build lsf

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/6867199311
